src/main/java/ru/shortly/controller/schemas/Link.java

```diff
@@ -1,46 +1,46 @@
 package ru.shortly.controller.schemas;
 
 public class Link {
-    private final String url;
-    private final String shortUrl;
+    private final ShortLink shortLink;
+    private final String longLink;
 
-    private Link(String url, String shortUrl) {
-        this.url = url;
-        this.shortUrl = shortUrl;
+    private Link(ShortLink shortLink, String longLink) {
```

Comment:
Maybe add null checks in constructor?Like Objects.requireNonNull(field,"field")?